app/views/hosts/_list.html.erb

```diff
@@ -32,6 +33,7 @@
         <td class="hidden-tablet hidden-xs ellipsis"><%= host.compute_resource_or_model %></td>
         <td class="hidden-tablet hidden-xs"><%= label_with_link host.hostgroup, 23, @hostgroup_authorizer %></td>
         <td class="hidden-tablet hidden-xs ellipsis"><%= last_report_column(host) %></td>
+        <%= render_pagelets_for(:hosts_table_content, :subject => host) %>
```

Comment:
Would `hosts_table_column` sound better?

Reply:
I changed both mountpoint names to `hosts_table_column_header` and `hosts_table_column_content`. Seems the most precise to me.